utils.groovy

```diff
@@ -706,4 +706,23 @@ def get_supported_additional_arches() {
     return supported
 }
 
+// try sending a message on matrix through maubot
+def matrixSend(message) {
+
+    withCredentials([usernamePassword(credentialsId: 'matrix-bot-webhook-token',
+                                      usernameVariable: 'MATRIX_WEBHOOK_URL',
+                                      passwordVariable: 'TOKEN')]) {
+
+    shwrap("""
```

Comment:
using shwrap here causes Jenkins to emit a warning because it would leak the secret. Hopefully nothing is leaked as Jenkins masks the secret : 
```
14:33:09  Masking supported pattern matches of $TOKEN
14:33:09  [Pipeline] {
14:33:09  [Pipeline] withEnv
14:33:09  [Pipeline] {
14:33:09  [Pipeline] sh
14:33:09  Warning: A secret was passed to "sh" using Groovy String interpolation, which is insecure.
14:33:09  		 Affected argument(s) used the following variable(s): [TOKEN]
```
I am still wondering if i should remove shwrap ?